### ISSUE TITLE
Fuse not updated after latest @vuseuse/integrations update

### ISSUE THREAD

**Issue by TheProcedural** [vueuse/vueuse#4049]
### Describe the bug

The reason behind the error below can be seen at line **161** here [https://www.npmjs.com/package/@vueuse/integrations?activeTab=code](https://www.npmjs.com/package/@vueuse/integrations?activeTab=code)

```bash
#npm error log

npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @vueuse/integrations@10.11.0
npm ERR! Found: fuse.js@7.0.0
npm ERR! node_modules/fuse.js
npm ERR!   fuse.js@"^7.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional fuse.js@"^6" from @vueuse/integrations@10.11.0
npm ERR! node_modules/@vueuse/integrations
npm ERR!   @vueuse/integrations@"^10.11.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: fuse.js@6.6.2
npm ERR! node_modules/fuse.js
npm ERR!   peerOptional fuse.js@"^6" from @vueuse/integrations@10.11.0
npm ERR!   node_modules/@vueuse/integrations
npm ERR!     @vueuse/integrations@"^10.11.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

### Reproduction

production

### System Info

```Shell
Not relevant
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vueuse/vueuse/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vueuse/vueuse/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vueuse.org/guide).
- [x] Check that there isn't [already an issue](https://github.com/vueuse/vueuse/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a VueUse issue and not a framework-specific issue. For example, if it's a Vue SFC related bug, it should likely be reported to https://github.com/vuejs/core instead.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vueuse/vueuse/discussions).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by pkc918** [comment]
It appears that the issue you've reported has been resolved in VueUse latest beta version([v11.0.0-beta.1](https://github.com/vueuse/vueuse/releases/tag/v11.0.0-beta.1)).

**Comment by TheProcedural** [comment]
A beta version that is not pointed out your docs:

[https://vueuse.org/integrations/useFuse/](https://vueuse.org/integrations/useFuse/)
![image](https://github.com/vueuse/vueuse/assets/54407345/1644c07c-2bea-48c4-b1c7-fae67feaa132)

[https://vueuse.org/integrations/README.html](https://vueuse.org/integrations/README.html)
![image](https://github.com/vueuse/vueuse/assets/54407345/d73ba197-efdc-4f0a-9399-f08711ca3f96)
